frontend/src/components/pages/CreateReview/BookModal.tsx

```diff
@@ -0,0 +1,275 @@
+                  name="illustrator-name"
+                  required={false}
+                  inputFields={illustrators}
+                  setInputFields={setIllustrator}
+                />
+                <AddStringInputList
+                  id="translator"
+                  label="Translator"
+                  name="translator-name"
+                  required={false}
+                  inputFields={translators}
+                  setInputFields={setTranslator}
+                />
+                <Stack align="center" spacing={5} isInline>
+                  <AddStringInput
+                    id="publishers"
+                    label="Publisher"
+                    name="publisher-name"
+                    required
+                    inputField={publisher}
+                    setInputField={setPublisher}
+                  />
+                  <FormControl id="Publication Year" isRequired>
+                    <FormLabel mb={2}>Publication Year</FormLabel>
+                    <AddNumberInput
+                      mb={2}
+                      placeholder="Year"
+                      setNumberField={setPublicationYear}
+                    />
+                  </FormControl>
+                </Stack>
+                <Stack align="center" spacing={5} isInline>
+                  <AddSelect
+                    id="formats"
+                    label="Format"
+                    required
+                    maxWidth="70%"
+                    values={Object.values(BookFormats)}
+                    setSelectField={setFormat}
+                  />
+                  <AddStringInput
+                    id="isbn"
+                    label="ISBN"
+                    name="isbn"
+                    required
+                    inputField={isbn}
+                    setInputField={setIsbn}
+                  />
+                  <FormControl id="price" isRequired width="45%">
+                    <FormLabel mb={2}>Price</FormLabel>
+                    <AddNumberInput
+                      mb={2}
+                      placeholder="$"
+                      setNumberField={setPrice}
+                    />
+                  </FormControl>
+                </Stack>
+              </Stack>
+            </GridItem>
+            <GridItem>
+              <Stack spacing={4} direction="column">
+                <AddStringInput
+                  id="bookCover"
+                  label="Book Cover"
+                  name="bookCover"
+                  placeholder="Image link here"
+                  required
+                  inputField={coverImage}
+                  setInputField={setCoverImage}
+                />
+                <AddSelect
+                  id="genre"
+                  label="Genre"
+                  required
+                  values={Object.values(Genre)}
+                  setSelectField={setGenre}
+                />
+                <FormControl isRequired width="100%">
```

Comment:
This is because most AddNumberInputs in the book modal doesn't have FormLabel, but if you think it's better to move it into the component, we can do that!